src/components/Drawer/Drawer.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { getIcons } from '../../util';
+import './Drawer.css';
+function Drawer() {
+  return (
+    <aside className="drawer">
+      <div className="drawer-icons">
+        {getIcons('HOME', '25px')}
+        <div className="text-center">Home</div>
+      </div>
+      <div className="drawer-icons">
+        {getIcons('HISTORY', '25px')}
+        <div className="text-center">History</div>
+      </div>
+      <div className="drawer-icons">
+        {getIcons('LIBRARY', '25px')}
+        <div className="text-center">Playlist</div>
+      </div>
+      <div className="drawer-icons">
+        {getIcons('WATCH_LATER', '25px')}
+        <div className="text-center">Watch Later</div>
+      </div>
```

Comment:
You could create a list having all the titles and just map over the `div` 4 times rather than repeating the same code 4 times. Try to look for `DRY` i.e. `Do not repeat` approach.